packages/zowe-explorer-ftp-extension/src/abstractFtpApi.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the *
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at *
+ * https://www.eclipse.org/legal/epl-v20.html                                      *
+ *                                                                                 *
+ * SPDX-License-Identifier: EPL-2.0                                                *
+ *                                                                                 *
+ * Copyright Contributors to the Zowe Project.                                     *
+ *                                                                                 *
+ */
+
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import * as imperative from "@zowe/imperative";
+import { ZoweExplorerApi } from "@zowe/zowe-explorer-api";
+
+export class AbstractFtpApi implements ZoweExplorerApi.ICommon {
```

Comment:
Add `abstract` before `class` to indicate it's an abstract class.

Reply:
done.